Microsoft.Toolkit.Uwp.UI.Media/Geometry/CanvasPathGeometry.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Numerics;
+using System.Text;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Brushes;
+using Microsoft.Graphics.Canvas.Geometry;
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Common;
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Parsers;
+using Windows.UI;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry
+{
+    /// <summary>
+    /// Helper Class for creating Win2d objects
+    /// </summary>
+    public static class CanvasPathGeometry
+    {
+        /// <summary>
+        /// Parses the Path data string and converts it to CanvasGeometry.
+        /// </summary>
+        /// <param name="resourceCreator">ICanvasResourceCreator</param>
+        /// <param name="pathData">Path data</param>
+        /// <param name="logger">(Optional) For logging purpose. To log the set of
+        /// CanvasPathBuilder commands, used for creating the CanvasGeometry, in
+        /// string format.</param>
+        /// <returns><see cref="CanvasGeometry"/></returns>
+        public static CanvasGeometry CreateGeometry(ICanvasResourceCreator resourceCreator, string pathData, StringBuilder logger = null)
```

Comment:
Since this is the public entry point, can we have a few unit tests around providing this some valid and invalid data and ensuring it behaves as expected?

Reply:
Added unit test cases with both valid and invalid data.